tests/demoQA/pages/inputs.js

```diff
@@ -0,0 +1,18 @@
+
+function randomizeInput() {
+    const input = Math.random().toString(36).substring(2, 10)
+    return input;
+}
+
+function randomizeEmail() {
+    const email = Math.random().toString(36).substring(2, 10) + '@mail.com'
+    return email;
```

Comment:
we can use just return `Math.random().toString(36).substring(2, 10) + '@mail.com'`